client.go

```diff
@@ -32,6 +32,10 @@ const (
 	closedSessionKeyCleanupInterval = time.Minute
 )
 
+var (
+	ErrTunaSessionClosed = fmt.Errorf("tuna session is closed")
```

Comment:
It's enough to call ErrClosed because it will be ts.ErrClosed from outside the package

Reply:
Yes, modified.